### PR TITLE
update readmes

### PR DESCRIPTION
**Pull request by imtoori**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,45 +24,28 @@ To get started, run `bootstrap` after cloning the project.
...
-melos run test:web
-```
+The best place to start is the [Flutter Chat Tutorial](https://getstream.io/chat/flutter/tutorial/).
+It teaches you how to use this SDK and also shows how to make frequently required changes.
+
+## Example Apps
+
+Every package folder includes a fully functional example with setup instructions.
+
+We also provide a set of sample apps created using the Stream Flutter SDK at this location. https://github.com/GetStream/flutter-samples
```
Comment (deven98) [inline README.md:51]: Could we make 'at this location' the link? If not, then we should probably make it 'at this location: https://...'

Comment (imtoori) [reply README.md:51]: makes sense

File: README.md [hunk 2]
```diff
@@ -15,45 +24,28 @@ To get started, run `bootstrap` after cloning the project.
...
+### [stream_chat_persistence](https://github.com/GetStream/stream-chat-flutter/tree/master/packages/stream_chat_persistence)
+This package provides a persistence client for fetching and saving chat data locally. Stream Chat Persistence uses Moor as a disk cache.
 
-### Build APK
-Builds an Android APK for examples
-```shell
-melos run build:examples:android
-```
+### [stream_chat_flutter_core](https://github.com/GetStream/stream-chat-flutter/tree/master/packages/stream_chat_flutter_core)
+This package provides business logic to fetch common things required for integrating Stream Chat into your application. The core package allows more customisation and hence provides business logic but no UI components.
```
Comment (Nash0x7E2) [inline README.md:37]: `Core` should be formatted as cod.

**Review by Nash0x7E2 — DISMISSED** [review]
Would be nice to link to the changelog for recent changes. 

LGTM

**Pull request merged**